### PR TITLE
fix: remove circular parent in ingredients taxonomy

### PR DESCRIPTION
**Pull request by stephanegigandet**
separate "fond" from "broth"

### CODE REVIEW

File: lib/ProductOpener/Tags.pm [hunk 1]
```diff
@@ -1058,7 +1058,7 @@ sub get_file_from_cache ($source, $target) {
 # Change this version string if you want to force the taxonomies to be rebuilt
 # e.g. if the taxonomy building algorithm or configuration has changed
 # This needs to be done also when the unaccenting parameters for languages set in Config.pm are changed
-my $BUILD_TAGS_VERSION = "20240329 - better taxonomy errors handling";
+my $BUILD_TAGS_VERSION = "20240403 - fix issue with additives.properties.txt not loaded + circular_parent check";
```
Comment (alexgarel) [inline lib/ProductOpener/Tags.pm:1061]: you should not need it if you touch ingredients.txt

This is only needed when the code change.

Comment (alexgarel) [reply lib/ProductOpener/Tags.pm:1061]:
```suggestion
my $BUILD_TAGS_VERSION = "20240329 - better taxonomy errors handling";
```

**Review by alexgarel — APPROVED** [review]
Thanks, but maybe we should not change Tags.pm variable.

**Comment by alexgarel** [comment]
Test failing:

```
2024-04-03T11:16:57.4811136Z [  FAIL  ]  job 59  + <UNNAMED ASSERTION>
2024-04-03T11:16:57.5015143Z (  DIAG  )  job 59      Failed test at tests/unit/tags.t line 715.
2024-04-03T11:16:57.5016275Z (  DIAG  )  job 59             got: 'en:colour,en:stabiliser'
2024-04-03T11:16:57.5017158Z (  DIAG  )  job 59        expected: 'en:colour, en:stabiliser'
2024-04-03T11:16:57.5881174Z (  DIAG  )  job 59    Looks like you failed 1 test of 195.
```

I though this was fixed !

**Comment by stephanegigandet** [comment]
> Test failing:
> 
> ```
> 2024-04-03T11:16:57.4811136Z [  FAIL  ]  job 59  + <UNNAMED ASSERTION>
> 2024-04-03T11:16:57.5015143Z (  DIAG  )  job 59      Failed test at tests/unit/tags.t line 715.
> 2024-04-03T11:16:57.5016275Z (  DIAG  )  job 59             got: 'en:colour,en:stabiliser'
> 2024-04-03T11:16:57.5017158Z (  DIAG  )  job 59        expected: 'en:colour, en:stabiliser'
> 2024-04-03T11:16:57.5881174Z (  DIAG  )  job 59    Looks like you failed 1 test of 195.
> ```
> 
> I though this was fixed !

@alexgarel That's because we still had the code in Tags.pm to canonicalize properties. I just removed it as it's now done in the linter:
https://github.com/openfoodfacts/openfoodfacts-server/pull/10084/commits/0490d032624f269be75392be1a5e7ed60e5c6ff3

File: lib/ProductOpener/Tags.pm (hunk 1 above)
Comment (alexgarel) [reply lib/ProductOpener/Tags.pm:1061]: ok, let's merge asap :-)

**Comment by stephanegigandet** [comment]
I forgot to change the version so that taxonomies are rebuilt... :-|

**Comment by alexgarel** [comment]
a tidy lint was also needed… done !

**Pull request merged**